rootless/rootless.go

```diff
@@ -5,22 +5,21 @@ import (
 	"sync"
 )
 
+const (
+	// RootlessKitDockerProxyBinary is the binary name of rootlesskit-docker-proxy
+	RootlessKitDockerProxyBinary = "rootlesskit-docker-proxy"
+)
+
 var (
-	runningWithNonRootUsername     bool
-	runningWithNonRootUsernameOnce sync.Once
+	runningWithRootlessKit     bool
+	runningWithRootlessKitOnce sync.Once
 )
 
-// RunningWithNonRootUsername returns true if we $USER is set to a non-root value,
-// regardless to the UID/EUID value.
-//
-// The value of this variable is mostly used for configuring default paths.
-// If the value is true, $HOME and $XDG_RUNTIME_DIR should be honored for setting up the default paths.
-// If false (not only EUID==0 but also $USER==root), $HOME and $XDG_RUNTIME_DIR should be ignored
-// even if we are in a user namespace.
-func RunningWithNonRootUsername() bool {
-	runningWithNonRootUsernameOnce.Do(func() {
-		u := os.Getenv("USER")
-		runningWithNonRootUsername = u != "" && u != "root"
+// RunningWithRootlessKit returns true if running under RootlessKit namespaces.
+func RunningWithRootlessKit() bool {
+	runningWithRootlessKitOnce.Do(func() {
+		u := os.Getenv("ROOTLESSKIT_STATE_DIR")
+		runningWithRootlessKit = u != ""
```

Comment:
Should we look for a truth value here such as 1 || true ?

Reply:
This is a directory path, not boolean.

https://github.com/rootless-containers/rootlesskit/blob/master/README.md#environment-variables

Reply:
Omg, early morning... don't mind me...

Reply:
@cpuguy83 good to go?

Reply:
Yes, didn't mark it since I was on a bus on my phone 😺